pyracf/common/logger.py

```diff
@@ -93,33 +94,63 @@ def log_debug(self, header_message: str, message: str) -> None:
         )
         print(f"{header}\n{message}")
 
-    def redact_strings(
+    def __redact_dict(
```

Comment:
I would change `__redact_dict` to `__redact_request_dictionary` dictionary since this only redacts values in a request dictionary.

Reply:
Agree, this is more descriptive and accurate. Change made